libindy/src/services/wallet/encryption.rs

```diff
@@ -0,0 +1,131 @@
+use std::collections::HashMap;
+
+use sodiumoxide::crypto::aead::xchacha20poly1305_ietf;
```

Comment:
All crypto must be wrapper in utils\crypto. Direct links to libsodium in code isn't allowed